### PR TITLE
Update readme SSL and 4.0 Operator

### PR DESCRIPTION
**Pull request by leszko**
This PR should be merged after:
- related[ Helm Chart PR](https://github.com/hazelcast/charts/pull/109) is merged
- new Helm Chart (`3.1.0`) is released
- new Operator (`0.2.0`) is generated and released

### CODE REVIEW

File: operator.yaml [hunk 1]
```diff
@@ -54,13 +54,13 @@ spec:
...
-              value: hazelcast/management-center:3.12.6
+              value: hazelcast/management-center:4.0
           resources:
             limits:
-              cpu: "1"
+              cpu: "0.1"
               memory: 512Mi
             requests:
-              cpu: "0.5"
+              cpu: "0.1"
```
Comment (mesutcelik) [inline operator.yaml]: How did you come up with those figures?

Comment (leszko) [reply operator.yaml]: We didn't do any measurements, I took them from some Kubernetes example. To be honest, they will depend on your environment, but `0.5` was definitely too high, you don't need that much to run operator.

**Comment by leszko** [comment]
@mesutcelik @kwart wrt Cert Manager. Currently I only described the general way of setting security for Hazelcast + mentioned that it works fine with Cert Manager. That is actually what was required for the IBM Certification.

I get your point that it would be nice to have the Cert Manager configuration with Hazelcast here, but I'm a little afraid of the maintenance effort. Like we described [Minishift in OpenShift Code Sample](https://github.com/hazelcast/hazelcast-code-samples/tree/master/hazelcast-integration/openshift) and nobody knows if the Minishift steps are valid any mode, because to check it you'd need to spend the whole day.

My idea is to:
1. Leave it as it is for now (it's good enough for the IBM certification)
2. Write a blog post describing Cert Manager + Hazelcast Operator configuration (I can write it, or Josef if you want, you can write it)
3. Link the blog post here in the operator description.

What do you think?

Another idea would be to remove this keystore/truststore description (from both Helm Chart and hazelcast-operator) and add this keys which are injected by Cert Manager, so `{ca.crt, tls.key, tls.crt}` instead of `{keystore, truststore}`.

**Comment by leszko** [comment]
> ... and some mention about Hazelcast 4.0 support version of the operator. Can you also update those or send separate RP?

Added versions compatibility section.

**Comment by mesutcelik** [comment]
I see Rafal. The reason I asked because I did not even see the mention to Certificate Manager. Now I see it. I think you can at least put a website link to https://cert-manager.io/docs/

**Comment by leszko** [comment]
> https://cert-manager.io/docs/

Updated with the link

**Pull request merged**